homeassistant/components/updater/binary_sensor.py

```diff
@@ -0,0 +1,62 @@
+"""Support for Home Assistant Updater binary sensors."""
+
+from homeassistant.components.binary_sensor import BinarySensorDevice
+
+from . import ATTR_NEWEST_VERSION, ATTR_RELEASE_NOTES, DOMAIN
+
+
+async def async_setup_platform(hass, config, async_add_entities, discovery_info=None):
+    """Set up the updater binary sensors."""
+    updater = hass.data[DOMAIN]
+
+    async_add_entities([UpdaterBinary(updater)])
+
+
+class UpdaterBinary(BinarySensorDevice):
+    """Representation of an updater binary sensor."""
+
+    def __init__(self, updater):
+        """Initialize the binary sensor."""
+        self._updater = updater
+
+    @property
+    def name(self) -> str:
+        """Return the name of the binary sensor, if any."""
+        return "Updater"
+
+    @property
+    def unique_id(self) -> str:
+        """Return a unique ID."""
+        return "updater"
+
+    @property
+    def is_on(self) -> bool:
+        """Return true if the binary sensor is on."""
+        return "on" if self._updater.update_available else "off"
+
+    @property
+    def device_class(self) -> str:
+        """Return the class of this device, from component DEVICE_CLASSES."""
+        return "connectivity"
```

Comment:
This shouldn't be `"connectivity"`. I don't think we have a device class that fits. We can remove this property.

Reply:
Done